roles/docker/tasks/main.yml

```diff
@@ -12,7 +12,7 @@
   register: start_result
 
 - set_fact:
-    docker_service_status_changed = start_result | changed
+    docker_service_status_changed: start_result | changed
```

Comment:
Good deal if it's firing on changes. I had checked a `debug: var=some_status_changed` and it was literally storing `"start_result | changed"` rather than `True` or `False`.